matrix.go

```diff
@@ -99,6 +99,7 @@ func (mx *WebsocketCommandHandler) handleWSSyncProxyError(cmd appservice.Websock
 type ProfileOverride struct {
 	Displayname string `json:"displayname,omitempty"`
 	PhotoURL    string `json:"photo_url,omitempty"`
+	PhotoMXC    string `json:"photo_mxc,omitempty"`
```

Comment:
I think this can just be `id.ContentURI` directly, that way you don't need to parse it manually

```suggestion
	PhotoMXC    id.ContentURI `json:"photo_mxc,omitempty"`
```